### DISCUSSION TITLE
.net Maui stopped working on VS2022 version 17.11.4

### DISCUSSION THREAD

**Discussion by alvaroseh** [CommunityToolkit/Maui#2233] [Q&A]
I am using .net Maui Blazor to develop Android and IOS applications. After updating my VS 2022 with the version 17.11.4, the Visual studio does not recognize anymore some commands that did before, for example @inject NavigationManager NavManager;, @code {
	[Parameter]
	public bool _isExpand { get; set; } = false; 
 What is happening?

**Comment by bijington** [comment]
I think you want to ask on the MAUI repo https://github.com/dotnet/maui. This repo is for a community based toolkit
